### DISCUSSION TITLE
Monthly Meeting: 5. November 2024, 2pm CET

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#2525] [Show and tell]
Participants: @KaiVolland @hwbllmnn @dnlkoch @jansule @ger-benjamin 

## TOPICS

- OSGeo fundings
  - funding period (geostyler code sprint)
  - incubation fund
  - @dnlkoch get information about how and when to apply for fundings (see also https://wiki.osgeo.org/wiki/OSGeo_Budget_2025)
- official OSGeo project
  - TODO @KaiVolland @jansule sit together and create application / contact incubation mentor
- FOSSGIS 2025
  - talk: latest changes + ArcGIS parser
  - demo-session: styling maps with GeoStyler
- symbol preview when using expressions
  - TODO @jansule create issue
- SLD vendor options (missing issue)
  - TODO @KaiVolland @ger-benjamin create issue (see also https://github.com/orgs/geostyler/discussions/2499)

## OTHER TODOs

- update architecture graphic (https://github.com/geostyler/geostyler/blob/main/docs/ComponentView.jpg)

**Comment by jansule** [comment]
Issue regarding the preview when using expressions created: https://github.com/geostyler/geostyler/issues/2526

**Comment by jansule** [comment]
OSGeo incubation mentor contacted

**Comment by KaiVolland** [comment]
Issue for SLD vendor options created: https://github.com/geostyler/geostyler-sld-parser/issues/969
